### ISSUE TITLE
Bug Report:  When receiving a rank for a song in freeplay menu, the inscriptions(BPM, week, difficulty and name of the song) and the opponent’s icon become unstuck from the animation.

### ISSUE THREAD

**Issue by BStasik** [FunkinCrew/Funkin#2755]
## Describe the bug
Inscriptions and icon look strange when receiving a rank for any of the songs in freeplay menu.

## To Reproduce
No special actions needed.

## Expected behavior
They must be attached to the animation.

## Screenshots/Video
![image](https://github.com/FunkinCrew/Funkin/assets/172075123/e30dff3d-e22d-4627-b85d-2cd1da9d6d2f)
![image](https://github.com/FunkinCrew/Funkin/assets/172075123/bda58bf2-fdc9-4df4-8a6d-0d51b0ef5d7e)
![image](https://github.com/FunkinCrew/Funkin/assets/172075123/4de3dd84-a71e-4fb7-8199-12df19e7bcc4)

## Desktop
 - OS:
Windows 10
 - Browser
Downloaded from itch.io
 - Version:
0.4.0

## Additional context
Screenshots are just examples.

**Comment by Hundrec** [comment]
I can't find any other issue reporting this or anything indicating when it was fixed.
If anyone can find another issue or knows when this was fixed, let me know and I'll label this issue accordingly.

**Comment by BStasik** [comment]
i dont think it was fixed but i also dont see it as a bug anymore, i guess it is intentional (i only could take screenshots by skipping per 1 frame on recorded gameplay)

close this issue if you agree with me

**Comment by Hundrec** [comment]
I haven't seen it in a long enough time to believe it's been fixed.

If anyone else encounters this issue, feel free to make a new one!
